GSG-K3/360HMR
# History front end

Call the history with http://localhost:3000/dashboard/history endpoint.

on this page you can see a list of employees that have been reviewed before

Does this Fix an Issue? Fixes #41

How to test this PR npm i

cd client && npm i

cd..

npm run dev

go to http://localhost:3000/dashboard/history endpoint

Screenshots of changes if the database is empty you will only see a page loading
https://ibb.co/1v0qyT6
